test/tools/Modules/HttpListener/HttpListener.psm1

```diff
@@ -133,6 +133,14 @@ Function Start-HTTPListener {
                             $statusCode = $queryItems["statuscode"]
                             $contentType = $queryItems["contenttype"]
                             $output = $queryItems["output"]
+                            if ($queryItems['headers'])
```

Comment:
It would be helpful to add a comment here that illustrates how the 'headers' value should be formatted; especially since this is the only documentation we have for the listener. :)  
At a minimum, show the json structure.  Even better would be to show a code sample similar to what you used in your tests.